pkg/store/forward/forward.go

```diff
@@ -109,7 +109,12 @@ func (s *Store) WriteMetrics(ctx context.Context, p *store.PartitionedMetrics) e
 				Observe(time.Since(begin).Seconds())
 
 			if resp.StatusCode/100 != 2 {
-				return fmt.Errorf("response was not 200 OK, but %s", resp.Status)
+				mean := timeseriesMeanDrift(timeseries, time.Now().Unix()*1000)
+				return fmt.Errorf("response status code is %s - mean drift from now for clusters %S is: %.3fms",
+					resp.Status,
+					p.PartitionKey,
+					float64(time.Now().Second())-mean,
```

Comment:
The mean drift will be something like `200ms`, we dont need to subtract that again from `now`, right?

Reply:
Exactly. I already calculate the mean of the drift (which is now - timestamp) per timestamp.